xml/System.Web.UI/HtmlTextWriter.xml

```diff
@@ -278,7 +278,7 @@ alert('Hello');
       <Docs>
         <param name="name">A string containing the name of the attribute to add.</param>
         <param name="value">A string containing the value to assign to the attribute.</param>
-        <param name="fEndode">
+        <param name="fEncode">
```

Comment:
Unfortunately, @Rick-Anderson, `fEndode` is the parameter name defined in metadata; it can't be changed to fEncode. (Doing that causes the parameter description to disappear from the build.)

Reply:
On MSDN, you added this info on a different overload. Note that there are two with a boolean parameter:
https://msdn.microsoft.com/en-us/library/cbax60cy(v=vs.110).aspx - fEndode - this member
https://msdn.microsoft.com/en-us/library/ms153285(v=vs.110).aspx - fEncode which is the one where you added these remarks